frontend/src/scenes/insights/insightVizDataLogic.test.ts

```diff
@@ -364,6 +369,34 @@ describe('insightVizDataLogic', () => {
                 },
             })
         })
+
+        it('clears smoothing ', async () => {
```

Comment:
For me, this test's name would benefit from explaining when smoothing should be cleared – I don't quite see why